lib/cadet/assessments/assessments.ex

```diff
@@ -5,6 +5,7 @@ defmodule Cadet.Assessments do
   """
   use Cadet, [:context, :display]
 
+  import Cadet.Chat.Room
```

Comment:
let's not import here. use `alias`

Reply:
Fixed with commit [3d3915b](https://github.com/source-academy/cadet/pull/406/commits/3d3915be3bd31d5b91b572ed2006f660ee2ed05c)